cmd/kyverno/main.go

```diff
@@ -138,14 +138,17 @@ func main() {
 	if err != nil {
 		setupLog.Error(err, "ConfigMap lookup disabled: failed to create resource cache")
 	}
+	debug := serverIP != ""
```

Comment:
This PR needs some changes, can we send a separate PR for the debug mode fix? @kacejot is blocking by it.

Reply:
Okay